PallindromPartioning.py

```diff
@@ -0,0 +1,40 @@
+# Time Complexity : O(2^N) Number of characters in string
+#  Space Complexity : O(N) Space taken by stack
+#  Did this code successfully run on Leetcode : Yes
+#  Any problem you faced while coding this : N/A
+
+#  Your code here along with comments explaining your approach
+#T.C  O(2^n)?
```

Comment:
No, It will be 2^n * n as over each character of given string, you are checking if it is palindrome or not, then you are doing recursion further so.